packages/components/src/styles/copyright/copyright.css

```diff
@@ -0,0 +1,8 @@
+.sd-copyright {
+  @apply relative;
+
+  &::after {
+    @apply block absolute p-4 bottom-0 left-0 text-white font-bold w-full content-[--copyright];
+    text-shadow: 1px 1px 2px black;
```

Comment:
Did you check this in design?

Reply:
oh very good question, this is a copy from your codePen sample. I just saw in Figma it's: `box-shadow: 0px 1px 3px 0px #515151BF;` but the color seems to be a manually set one. @solid-design-system/core-design which one should I use?